### PR TITLE
remove latest-release-notes from indexation

### PR DESCRIPTION
**Pull request by bemble**
## Proposed change

Right now, `/latest-release-notes/` is a simple HTML page that redirect to the latest release note. This page is included in the sitemap and Google indexation bot goes through this page. We recently had an indexation issue, where the selected `/latest-release-notes/` as canical URL instead of the right one (the link is in the "Recent Posts" section given page): 
Capture d’écran 2023-04-17 à 16 22 19

Usage of `/latest-release-notes/` should be avoided:
- on the `index.html`, release note link points directly to the latest core blog post
- the link in different blog post has been changed to the corresponding link
- the redirection is now done by netlify in the `_redirect` (status `302`)
- `/latest-release-notes/` is not in the `sitemap.xml` anymore

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_posts/2020-02-19-community-highlights.markdown [hunk 1]
```diff
@@ -66,7 +66,7 @@ mysql -D homeassistant -e "SELECT entity_id, state, last_updated FROM states LIM
 
 ## Beta time!
 
-Today we are releasing the first beta of Home Assistant Core 0.106. It is packed with awesome features. For a sneak peek of what is coming, check the [beta release notes](https://rc.home-assistant.io/latest-release-notes/).
+Today we are releasing the first beta of Home Assistant Core 0.106. It is packed with awesome features. For a sneak peek of what is coming, check the [beta release notes](https://rc.home-assistant.io/blog/2020/02/26/release-106/).
```
Comment (frenck) [inline source/_posts/2020-02-19-community-highlights.markdown]: There is no point in linking to the beta website anymore, let's just point it at the current stable one

```suggestion
Today we are releasing the first beta of Home Assistant Core 0.106. It is packed with awesome features. For a sneak peek of what is coming, check the [beta release notes](/blog/2020/02/26/release-106/).
```

File: source/_posts/2020-03-15-community-highlights.markdown [hunk 2]
```diff
@@ -34,7 +34,7 @@ setup has a total of 5 Home Assistant instances, controlling 7x marine aquariums
 Those are some big views Kevin! I bet you will be happy with the new Lovelace
 features that are coming in Home Assistant Core 0.107. It will help
 you to trim down that a bit, into multiple Lovelace Dashboards (now in
-[beta, sneak peak?](https://rc.home-assistant.io/latest-release-notes/)).
+[beta, sneak peak?](https://rc.home-assistant.io/blog/2020/03/18/release-107/)).
```
Comment (frenck) [inline source/_posts/2020-03-15-community-highlights.markdown]:
```suggestion
[beta, sneak peak?](/blog/2020/03/18/release-107/)).
```

File: source/index.html [hunk 3]
```diff
@@ -16,12 +17,10 @@ <h1>
           site.current_minor_version }}.{{ site.current_patch_version }}
         </h1>
         Released:
-        <span class="release-date"
-          >{{ site.date_released | date: "%B %e, %Y" }}</span
-        >
+        <span class="release-date">{{ site.date_released | date: "%B %e, %Y" }}</span>
```
Comment (frenck) [inline source/index.html]: Unrelated change?

File: source/index.html [hunk 4]
```diff
@@ -47,10 +46,7 @@ <h1>Join The Community</h1>
         </div>
       </div>
 
-      <a
-        class="material-card highlight-blog-post"
-        href="/blog/2016/01/19/perfect-home-automation/"
-      >
+      <a class="material-card highlight-blog-post" href="/blog/2016/01/19/perfect-home-automation/">
```
Comment (frenck) [inline source/index.html]: Everything below here in this file seems to be unrelated?

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/index.html (hunk 3 above)
Comment (bemble) [reply source/index.html]: It's an automated code reformat on save, but nothing related to this PR directly yes.

File: source/index.html (hunk 4 above)
Comment (bemble) [reply source/index.html]: Same as above.

File: source/index.html (hunk 3 above)
Comment (frenck) [reply source/index.html]: Those have caused issues on this website. Please revert this unrelated change.

**Review by frenck — CHANGES_REQUESTED** [review]
Please rever the unrelated changes.

**Review by frenck — APPROVED** [review]
Thanks, @bemble 👍 

../Frenck

**Pull request merged**